netmiko/cisco/__init__.py

```diff
@@ -1,21 +1,14 @@
 from __future__ import unicode_literals
 from netmiko.cisco.cisco_ios import CiscoIosBase, CiscoIosSSH, CiscoIosTelnet, CiscoIosSerial
 from netmiko.cisco.cisco_ios import CiscoIosFileTransfer
-from netmiko.cisco.cisco_ios import InLineTransfer
 from netmiko.cisco.cisco_asa_ssh import CiscoAsaSSH, CiscoAsaFileTransfer
 from netmiko.cisco.cisco_nxos_ssh import CiscoNxosSSH, CiscoNxosFileTransfer
-from netmiko.cisco.cisco_xr_ssh import CiscoXrSSH
-from netmiko.cisco.cisco_ios import CiscoIosBase, CiscoIosSSH, CiscoIosTelnet
-from netmiko.cisco.cisco_asa_ssh import CiscoAsaSSH
-from netmiko.cisco.cisco_nxos_ssh import CiscoNxosSSH
```

Comment:
why are removing nxos, asa?

Reply:
This is a part of the upgrade.. I am trying to restore